.rubocop.yml

```diff
@@ -49,6 +49,12 @@ RSpec/DescribeClass:
 RSpec/ExampleLength:
   Enabled: false
 
+# FIXME: Find a better way to block output during specs, and fix relevant examples.
+RSpec/ExpectOutput:
```

Comment:
RSpec doesn't really provide a way to block output without making an assertion about it. There is some code in Reek that emits a warning and does other things. Our current setup is intended to be able to spec the other things while preventing those warnings from polluting the spec output.